packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx

```diff
@@ -99,20 +97,22 @@ export const CollectionNfts: FC = () => {
             mx="3"
             py={6}
           >
-            {collection.nfts.map((nft) => (
-              <NftFigure
-                key={`${nft.collectionId}-${nft.id}`}
-                onClick={() =>
-                  navigate(routes.sendNft(nft.collectionId, nft.id))
-                }
-              >
-                <NftItem
-                  thumbnailSrc={nft.metadata.image || ""}
-                  name={
-                    nft.metadata.name ||
-                    "Untitled"
-                  }
-                />
+            {collection.nftIds.map((nftId) => (
+              <NftFigure key={`${collectionId}-${nftId}`}>
+                <ErrorBoundary fallback={<H4 mt="10" textAlign="center">Deprecated NFT</H4>}>
```

Comment:
Sounds great :+1:

Reply:
The current process for obtaining NFT is:

1. First get all NFT token IDs
2. Get collection ids and collection metadata based on NFT ids, we cannot judge if the collection is deprecated based on the collection metadata
3. When the user enters the `CollectionNfts` page, the NFT metadata will be obtained. Only then will we know if NFTs are deprecated

So in the old version, if I had two deprecated NFTs in one collection, I could see there are two NFTs on the `AccountCollections` page, but if I clicked in, the NFTs don't show up because it's deprecated.

I had previously thought that we could also display deprecated NFTs, but currently our SDK only supports getting NFTs of the latest standard.

Reply:
I see, thanks for the explanation. Since deprecated NFTs should not be on mainnet, I think it is okey to have them shown as deprecated and see if we receive some feedbacks